pkgs/applications/video/obs-studio/Add-full-support-for-CEF-4638.patch

```diff
@@ -0,0 +1,528 @@
+From b1a7f61a5f9418cee28e90e0fc07b47df017f58f Mon Sep 17 00:00:00 2001
```

Comment:
Will this patch ship in OBS 27.2? If so, it might make sense to wait for #154079 to get in?

Reply:
It is included in the latest release candidate. I guess we can wait for 27.2.

Reply:
:+1: